book/src/snapshot-controller.md

```diff
@@ -15,13 +15,12 @@ Latest stable release | Branch | Min CSI Version | Max CSI Version | Container I
 [external-snapshotter v4.2.1](https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v4.2.1) | [release-4.2](https://github.com/kubernetes-csi/external-snapshotter/tree/release-4.2) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | k8s.gcr.io/sig-storage/snapshot-controller:v4.2.1 | v1.20 | - | v1.22
 [external-snapshotter v4.1.1](https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v4.1.1) | [release-4.1](https://github.com/kubernetes-csi/external-snapshotter/tree/release-4.1) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | k8s.gcr.io/sig-storage/snapshot-controller:v4.1.1 | v1.20 | - | v1.20
 [external-snapshotter v4.0.0](https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v4.0.0) | [release-4.0](https://github.com/kubernetes-csi/external-snapshotter/tree/release-4.0) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | k8s.gcr.io/sig-storage/snapshot-controller:v4.0.0 | v1.20 | - | v1.20
```

Comment:
We just released v4.0.1 yesterday.  Can you update 4.0.0 to 4.0.1 for snapshot-controller, snapshot-validation-webhook, and csi-snapshotter sidecar?